cmd/performance-profile-creator/README.md

```diff
@@ -0,0 +1,55 @@
+# Performance Profile Creator (PPC)
+
+A tool to automate the process of creating Performance Profile using the user supplied profile parameters.
+
+## Software Components
+
+1. A CLI tool part of the PAO image
+1. A wrapper script <!--(TODO: Add link to where the wrapper script is placed) --> automates the access to PPC tool. It pulls the PAO image, supplies arguments and runs the tool inside Performance Addon Operator container with performance-profile-creator entrypoint.
+
+## Flow
+1. The user supplies the command line args to the tool by using a wrapper script.
+1. The wrapper triggers the must-gather tool.
+1. A must-gather output file is passed to PPC as an argument.
+1. The wrapper decompresses the must-gather output file and “mount” the ghw snapshot in order to inspect the hardware, and passes the snapshot to PPC. The ghw snapshots can also be supplied by the user may directly.
+1. PPC reuses decompressed ghw images and mount local dir for storage.
+1. Finally, PPC creates a PAO profile and optionally an NTO tuned overriding the PAO profile as yaml files. As an optional debug functionality and to enable the user to make further tuning decisions, it also outputs the interim queries and results.
+
+<!--
+## Wrapper Setup/Configuration
+ Add Steps to execute the wrapper script
+-->
+
+## Building Performance Profile Creator binary and image
+
+NOTE: Performance Profile Creator is present as an entrypoint (in /usr/local/bin/performance-profile-creator) in the Performance Addon Operator image.
+
+Developers can build the Performance Profile Creator images from the source tree using make targets.
+ 1. Setup Environment variables
+    ```bash
+    export REGISTRY_NAMESPACE=<your quay.io namespace>
+    export IMAGE_TAG=<the image tag to use> #defaults to "latest"
+    export IMAGE_BUILD_CMD=podman
+    ```
+1. To build from Performance Profile Creator source:
+   ```bash
+   make create-performance-profile
+   ```
+1. To build the Performance addon Operator image with  from source:
+   ```bash
+   make operator-container
+   ```
+Alternatively, you can pull the latest master upstream image.  In the following examples, TAG has the format major.minor-snapshot. For example, the TAG for OpenShift 4.8 will be 4.8-snapshot:
+
+```bash
+podman pull quay.io/repository/openshift-kni/performance-addon-operator:<TAG>
```

Comment:
do we need the "repository" ? also I would use the tag you mentioned above, 4.8-snapshot

Reply:
Ah! My bad, will update.